stdlib/LinearAlgebra/src/eigen.jl

```diff
@@ -428,7 +496,19 @@ function eigmin(A::Union{Number, AbstractMatrix};
     minimum(v)
 end
 
-inv(A::Eigen) = A.vectors * inv(Diagonal(A.values)) / A.vectors
+"""
+    spectral(f, F::Eigen)
```

Comment:
This name feels confusing, how about `matrix_function`?

Reply:
Agreed.